packages/server/src/app/db/content/content.service.ts

```diff
@@ -1,10 +1,111 @@
-import { Injectable } from '@nestjs/common';
+import { BadRequestException, Injectable } from '@nestjs/common';
 import { InjectModel } from '@nestjs/mongoose';
-import { PaginateModel } from 'mongoose';
+import { JwtPayload } from '@pulp-fiction/models/auth';
+import { UserInfo } from '@pulp-fiction/models/users';
+import { AnyKindOfDictionary } from 'lodash';
+import { PaginateModel, PaginateResult } from 'mongoose';
 
 import { ContentDocument } from './content.schema';
 
 @Injectable()
 export class ContentService {
     constructor(@InjectModel('Content') private readonly contentModel: PaginateModel<ContentDocument>) {}
+
+    /**
+     * Fetches one item from the content collection via ID.
+     * 
+     * @param contentId A content's ID
+     */
+    async fetchOneUnpublished(contentId: string) {
+        return await this.contentModel.findById(contentId).where('audit.isDeleted', false);
+    }
+
+    /**
+     * Fetches one published item from the content collection via ID.
+     * 
+     * @param contentId A content's ID
+     * @param kind A content's Kind
+     */
+    async fetchOnePublished(contentId: string, kind: string, user?: JwtPayload) {
+        if (kind === 'NewsContent') {
+            const post = await this.contentModel.findOne({'_id': contentId, 'kind': kind, 'audit.isDeleted': false, 'audit.published': true});
+            if (user) {
+                await this.addView(user, (post.author as any)._id, post._id);
+                return post;
+            } else {
+                return post;
+            }
+        } else if (kind === 'WorkContent') {
+            // find work doc
+        } else if (kind === 'BlogContent') {
+            //find blog doc
+        } else {
+            throw new BadRequestException(`The designated document kind does not exist.`);
+        }
+    }
+
+    /**
+     * Finds a bunch of content filtered by kind
+     * 
+     * @param kind The kind of doc
+     * @param pageNum The page number to grab
+     * @param published Optional parameter, checks whether item needs to be published
+     */
+    async fetchManyByKind(kind: string, pageNum: number, published?: boolean):  Promise<PaginateResult<ContentDocument>> {
+        if (published) {
+            if (published === true) {
```

Comment:
I think you can simplify this if-chain to just 
```typescript
if (published === true) { ....
```
...because if `published` is null or `undefined`, that'll evaluate false anyway.